src/core/mixins/courseFiltersMixin.js

```diff
@@ -26,7 +30,8 @@ export const courseFiltersMixin = {
         courses = courses.filter(course => course.program._id === this.selectedProgram);
       }
       if (this.selectedTrainer) {
-        courses = courses.filter(course => course.trainer && course.trainer._id === this.selectedTrainer);
+        courses = courses.filter(course => (this.selectedTrainer === 'without_trainer' && !course.trainer) ||
+          (course.trainer && course.trainer._id === this.selectedTrainer));
```

Comment:
on pourrait pas faire " course.trainer ? this.selectedTrainer === course.trainer._id : this.selectedTrainer === 'withoutTrainer' " ?

Reply:
Si tu as raison :)